client/footer.ejs

```diff
@@ -1,5 +1,12 @@
-<footer flex="none">
-  <a href="/about">About</a>
-  <a href="/tos">Terms</a>
-  <a href="/privacy-policy">Privacy</a>
+<footer layout="row">
+  <span flex class="footer-left">
+    <a href="/about">About</a>
+    <a href="/tos">Terms</a>
+    <a href="/privacy-policy">Privacy</a>
+  </span>
+  <span flex></span>
+  <span flex class="footer-right">
+    <a href="https://github.com/porygonco/porybox">GitHub</a> |
```

Comment:
I think it's already implied that the reason the link is there is because it goes to our repo. 

It's pretty common to just show Twitter, Facebook, etc. icons and the implication is that it goes to that site's profile on those pages.